ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -252,9 +250,7 @@ def test_source_ipahealthcheck_meta_services_check_sssd(self):
         """
         self.master.run_command(["systemctl", "stop", "sssd"])
         returncode, data = run_healthcheck(
-            self.master,
-            "ipahealthcheck.meta.services",
-            "sssd",
+            self.master, "ipahealthcheck.meta.services", "sssd",
         )
```

Comment:
Is there a reason this unrelated reformatting is included?

Reply:
@rcritten actually this was done by the tool i had used to format, will revert the changes.